egs/wsj/s5/steps/libs/nnet3/train/frame_level_objf/common.py

```diff
@@ -95,8 +101,7 @@ def train_new_models(dir, iter, srand, num_jobs,
             """ark:{egs_dir}/egs.{archive_index}.ark ark:- |"""
             """nnet3-shuffle-egs --buffer-size={shuffle_buffer_size} """
             """--srand={srand} ark:- ark:- | """
-            """nnet3-merge-egs --minibatch-size={minibatch_size} """
-            """--measure-output-frames=false """
+            """nnet3-merge-egs {minibatch_opts} """
```

Comment:
Er-- wait. 
This PR wouldn't have made any difference to any setup.
If chunk_level_training is false, then at this point the egs have only one output frame per eg.  So in that case it doesn't matter what value the config has.  In fact, there are no setups now in which we need --measure-output-frames=true, so we might as well always set it false to keep the script simple.
I'm wondering whether to revert this.